### PR TITLE
Update update-a-web-application-url-and-iis-bindings.md

### PR DESCRIPTION
**Pull request by msbemba**
Replaced the invlid link with https://docs.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/update-a-web-application-url-and-iis-bindings.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ If you are using a load balancer or a reverse proxy, make sure that your interna
   
 ### Apply an SSL certificate
 
-If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone. For more information about how to configure SSL, see [Configuring Secure Sockets Layer (IIS 6.0)](http://go.microsoft.com/fwlink/p/?LinkId=110300&amp;clcid=0x409).
+If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone. For more information about how to configure SSL, see [Configuring Secure Sockets Layer (IIS 7.0)](https://docs.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/update-a-web-application-url-and-iis-bindings.md]:
```suggestion
If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone.  See [How to Set Up SSL on IIS 7)](https://docs.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis) for more information about configuring SSL.
```

File: SharePoint/SharePointServer/administration/update-a-web-application-url-and-iis-bindings.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ If you are using a load balancer or a reverse proxy, make sure that your interna
   
 ### Apply an SSL certificate
 
-If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone. For more information about how to configure SSL, see [Configuring Secure Sockets Layer (IIS 6.0)](http://go.microsoft.com/fwlink/p/?LinkId=110300&amp;clcid=0x409).
+If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone.  See [How to Set Up SSL on IIS 7)](https://docs.microsoft.com/en-us/iis/manage/configuring-security/how-to-set-up-ssl-on-iis) for more information about configuring SSL.
```
Comment (nenonix) [inline SharePoint/SharePointServer/administration/update-a-web-application-url-and-iis-bindings.md]:
```suggestion
If the new IIS bindings use SSL, apply an SSL certificate to the new IIS website assigned to your zone.  See [How to Set Up SSL on IIS 7)](https://docs.microsoft.com/iis/manage/configuring-security/how-to-set-up-ssl-on-iis) for more information about configuring SSL.
```

**Comment by scanum** [comment]
@Techwriter40 Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**